src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Controls/UnoWebView/UnoWebView.iOS.cs

```diff
@@ -12,23 +12,24 @@ partial class UnoWebView
 		{
 			NSUrlCache.SharedCache.RemoveAllCachedResponses();
 
-			if (UIDevice.CurrentDevice.CheckSystemVersion(11, 0))
+			var CookieStorage = NSHttpCookieStorage.SharedStorage;
+			foreach (var cookie in CookieStorage.Cookies)
 			{
-				WKWebsiteDataStore.DefaultDataStore.HttpCookieStore.GetAllCookies((cookies) =>
-				{
-					foreach (var cookie in cookies)
-					{
-						WKWebsiteDataStore.DefaultDataStore.HttpCookieStore.DeleteCookie(cookie, null);
-					}
-				});
-			}
-			else
-			{
-				foreach (var cookie in NSHttpCookieStorage.SharedStorage.Cookies)
-				{
-					NSHttpCookieStorage.SharedStorage.DeleteCookie(cookie);
-				}
+				CookieStorage.DeleteCookie(cookie);
 			}
+
+			// For unknown reason, it seems that this does not delete correctly cookies (ios >= 11)
+			// TODO need to investigate why
```

Comment:
You might want to add a task or issue number somewhere for that TODO so that it's not forgotten :)